changelog/0007-reorganize-pillows.yml

```diff
@@ -0,0 +1,107 @@
+  FormSubmissionMetadataTrackerPillow | get_ucr_es_form_pillow
+  kafka-ucr-main | get_ucr_es_case_pillow, get_ucr_es_form_pillow, get_user_es_ucr_pillow
+  AppDbChangeFeedPillow | No Change
+  ApplicationToElasticsearchPillow | No Change
+  CacheInvalidatePillow | No Change
+  DefaultChangeFeedPillow | No Change
+  DomainDbKafkaPillow | No Change
+  FarmerRecordFluffPillow | No Change
+  GeographyFluffPillow | No Change
+  GroupPillow | get_group_es_pillow
+  GroupToUserPillow | get_group_es_pillow
+  KafkaDomainPillow | No Change
+  LedgerToElasticsearchPillow | No Change
+  M4ChangeFormFluffPillow | No Change
+  ReportCaseToElasticsearchPillow | get_ucr_es_case_pillow
+  ReportXFormToElasticsearchPillow | get_ucr_es_form_pillow
+  SqlSMSPillow | No Change
+  UCLAPatientFluffPillow | No Change
+  UnknownUsersPillow | get_ucr_es_form_pillow
+  UpdateUserSyncHistoryPillow | No Change
+  UserCacheInvalidatePillow | No Change
+  UserGroupsDbKafkaPillow | No Change
+  UserPillow | get_user_es_ucr_pillow  (includes user UCR processor)
+
+update_steps: |
+  Ensure that you have deployed a version of CommCare later than the version mentioned above
+  and that none of pillows for a given document type are too far behind from each other.
+
+  1. Replace old pillows with new pillows in pillow section of your app_processes.yml
+     Here's an example [pull request](https://github.com/dimagi/commcare-cloud/pull/2415)
+     Do not merge this until the switch is fully complete. Make sure you create enough
+     processes to handle your load.
+  2. Stop all pillows. You may have to be on above branch (for e.g. pillow-reorg)
+  ```bash   
+  cchq <env> fab stop_pillows --branch=pillow-reorg
+  ```
+  3. Run below django management command to create checkpoints for new pillows.
+  ```bash   
+  cchq <env> django-manage create_checkpoints_for_merged_pillows
+  ```
+  You can pass `--check` option to dry-run.
+  4. Deploy the change so that supervisor configuration is updated for the pillow switch.
+  ```bash   
+  cchq <env> ap deploy_commcarehq.yml --tags=services --limit=pillowtop --branch=pillow-reorg
+  ```
+  5. Run below command so that the supervisor process rereads the new config
+  ```bash   
+  cchq <env> update-supervisor-confs --branch=pillow-reorg
+  ```
+  6. Finally start all pillows.
+  ```bash    
+  cchq <env> fab start_pillows --branch=pillow-reorg
+  ```
+
+  You may do few checks to make sure the change is deployed succesfully.
+
+  1. Do a `sudo supervisorctl status all` on all pillow machines and make sure that new pillows are running succesfully and old pillows are not.
+  2. Look at pillow logs for new pillows and make sure there are no errors.
+  3. Do basic QA of various change feeds like
+     - Submit few forms, and make sure they appear in the UCR and standard reports
+     - Create few mobile workers and make sure they are searchable in report filters
+  4. Keep an eye on https://app.datadoghq.com/dash/256236/change-feeds-pillows
```

Comment:
Should the changelog be written assuming the audience is a non-Dimagi reader? If so, then this should be removed and instead point to the relevant part of `/hq/admin/system` as non-Dimagi readers don't have access to our datadog dashboards

Reply:
Yeah, good point. I will update